src/wiki/getting-started/installing-optifine.md

```diff
@@ -6,30 +6,34 @@ eleventyNavigation:
 ---
 # Preamble
 
-You should use optifine alternatives when possible. [See the wiki page about it](../install-of-alternatives)
-# Installing optifine
+Because Optifine is an old, closed-source project, it evolved to cause many mod incompatibilities, and doesn't do as good of a job as some alternatives out there. 
+
+Also, if you're using the Fabric mod loader, there's no native support for it, and the current [workaround](#fabric) is known to be buggy and unstable due to the very nature of Optifine and Optifabric. 
 
+Therefore, you should use optifine alternatives whenever possible. [See the wiki page about it](../install-of-alternatives).
+
+# Installing optifine
 Make sure you know how to [download-mods](../download-mods) before attempting to install optifine.
 
-go to https://optifine.net/downloads and find your respective minecraft version eg. 1.18.1.
+Go to https://optifine.net/downloads and find your respective minecraft version (eg. 1.18.2).
 
-Once found click mirror. Now click the download button to download your optifine.jar
+Once found, click **mirror**. Now click the **download** button to download your optifine jar file.
 
-Remember where you have kept your .jar and continue to the PolyMC launcher follow steps from [download-mods](../download-mods) to get to version then choose either forge or fabric([!warning](#fabric)).
+Remember where you have kept your .jar and continue to the PolyMC launcher. Follow the steps from [download-mods](../download-mods) and choose either forge or fabric([!warning](#fabric)).
 
 ## Forge
-Forge does not require exta steps besides adding the .jar for optifine. 
+Forge does not require exta steps besides [adding the .jar for optifine](#adding-.jar-file). 
 
 NOTE: some OF versions **don't** work on Forge!
 
 ## Fabric 
 [Requires Fabric API](../download-mods/#fabric)
```

Comment:
Optifabric doesn't require FAPI but whatever

Reply:
Well they do say this in their wiki:
![image](https://user-images.githubusercontent.com/9145768/157509015-f7120790-a079-4aaf-98a5-4e7011e3fdd2.png)
Though it is from 2019 so idk

Reply:
2019 Optifabric was pretty different
Not listed as a dependency on cf nor on https://github.com/Chocohead/OptiFabric/blob/llama/src/main/resources/fabric.mod.json

Reply:
I'll just add a warning for versions < 1.16 then, as some people might run into problems without it

Reply:
ok then